addons/hr/models/hr_department.py

```diff
@@ -236,3 +236,10 @@ def get_department_hierarchy(self):
         }
 
         return hierarchy
+
+    def get_all_members(self):
```

Comment:
```suggestion
    def _get_all_members(self):
```
I think that the guideline is to make functions private (not callable by RPC) unless you use them that way